DemoBookApp/Views/Author/Delete.cshtml

```diff
@@ -1,6 +1,7 @@
 ﻿@model DemoBookApp.Models.AuthorModel
 @{
     ViewData["Title"] = "Delete";
+    var temp = (Model.Gender == 1 ? "Male" : "Female");
```

Comment:
can we map from enum?

Reply:
Yes, a simple type casting would do instead of comparison. Shall I implement that?